README.markdown

```diff
@@ -1294,6 +1330,54 @@ if (name == "Hello") {
 
 ```
 
+ ## Nested Arguments and Elements
+
+ Nested arguments and elements, such as can occur at function callsites or within container definitions, can make for difficult to read code. For such statements, you can use the following strategy. Add a newline after each opening parenthesis, bracket, or brace, and place each argument or element on its own line. Use indentation to indicate association with each level of the containing function calls or containers. Consider the following view models: 
+ 
+
+```swift
+struct CellViewModel {
+    let title: String
+    let detail: String
+    let showDisclosureIndicator: Bool
+}
+
+struct SectionViewModel {
+    let title: String
+    let dataSource: [CellViewModel]
+}
+
+```
+**Preferred:**
+
+```swift
+let firstSection = SectionViewModel(
+    title: "Splines",
+    dataSource: [
+        CellViewModel(
+            title: "Foo",
+            detail: "Reticulated",
+            showDisclosureIndicator: true
+        ),
+        CellViewModel(
+            title: "Bar",
+            detail: "Unreticulated",
+            showDisclosureIndicator: false
+        )
```

Comment:
True, though I can go either way on making that part of the style. I wish trailing commas were also available for function parameters/arguments for the sake of consistency when using this code organization strategy.